tasks/import-listings/README.md

```diff
@@ -0,0 +1,25 @@
+# Listings Import Task
+
+**This version is only intended to provide basic documentation**  
+**Future updates will expand on this**
+
+## Environment Variables
+
+| Name                       | Description                                                       | Example                                         | Default        | Type                   |
+| -------------------------- | ----------------------------------------------------------------- | ----------------------------------------------- | -------------- | ---------------------- |
+| DATABASE_URL               | The URI for the database to import listings into                  | postgres://bloom-dev:bloom@localhost:5432/bloom |                | string                 |
+| EXTERNAL_API_BASE          | The URL base (proto + host) for the backend to pull listings from | https://api.bloom.example                       |                | string                 |
+| LISTINGS_ENDPOINT_PATH     | The path to the listings endpoint                                 | /listings                                       | /listings      | string                 |
+| JURISDICTION_ENDPOINT_PATH | The path to the jurisdictions endpoint                            | /jurisdictions                                  | /jurisdictions | string                 |
+| JURISDICTION_INCLUDE_LIST  | The names of jurisdictions to import listings from                | San Jose,San Mateo,Alameda                      |                | comma-delimited string |
+| LISTING_VIEW               | The listing view to request from the endpoint                     | base                                            | base           | "base" \| "full"       |
+
+## Example Import Command
+
+```
+DATABASE_URL=postgres://bloom-dev:bloom@localhost:5432/bloom \
+EXTERNAL_API_BASE=https://api.housingbayarea.bloom.exygy.dev \
```

Comment:
@chriscasto do we want to add a note somewhere in the README as to which API_BASE is which for Exygy, so we have it as a point of reference?  For example, if someone wants to get the production listings vs. dev listings what should they use? And maybe provide caution as to why we'd only want to use production from time to time.

Reply:
That might be warranted, but remember that we're walking a very fine line here between "intended specifically for Doorway" and "completely open for public use and modification".  I'm not sure whether Exygy wants that information to be officially published, although I do acknowledge that much of it is scattered around the code anyway, just not in an "official and documented" capacity.